src/types/video.ts

```diff
@@ -211,7 +213,7 @@ export type ControlsStyles = {
 
 export interface ReactVideoProps extends ReactVideoEvents, ViewProps {
   source?: ReactVideoSource;
-  drm?: Drm;
+  drm?: Drm; // deprecated
```

Comment:
nit;
```
/** @deprecated */
drm?: Drm;
```
I think above comment style is better to let people know
(This comment makes to display line-through style on IDE)

Reply:
Let's leave it - I am planning on adding much more docs via JSDoc in code for better DevX

Reply:
I did it ! thank you @YangJonghun !